citation-matching/citation-matching-core-code/src/main/java/pl/edu/icm/coansys/citations/CoreCitationMatchingSimpleFactory.java

```diff
@@ -37,24 +43,34 @@ public CoreCitationMatchingService createCoreCitationMatchingService(JavaSparkCo
     
     //------------------------ PRIVATE --------------------------
     
-    private List<Pair<MatchableEntityHasher, MatchableEntityHasher>> createMatchableEntityHashers(List<String> hashGeneratorClassNames) throws InstantiationException, IllegalAccessException, ClassNotFoundException {
+    private List<Pair<MatchableEntityHasher, MatchableEntityHasher>> createDefaultMatchableEntityHashers() {
```

Comment:
I would rename it to just createMatchableEntityHashers. The 'default' word suggests that it is possible to set custom hashers somewhere else.